internal/mode/static/nginx/conf/nginx-plus.conf

```diff
@@ -55,18 +55,17 @@ http {
 }
 
 stream {
-	variables_hash_bucket_size 512;
-    variables_hash_max_size 1024;
+  variables_hash_bucket_size 512;
+  variables_hash_max_size 1024;
 
-    map_hash_max_size 2048;
-    map_hash_bucket_size 256;
+  map_hash_max_size 2048;
+  map_hash_bucket_size 256;
 
-    log_format stream-main '$remote_addr [$time_local] '
+  log_format stream-main '$remote_addr [$time_local] '
                               '$protocol $status $bytes_sent $bytes_received '
                               '$session_time "$ssl_preread_server_name"';
```

Comment:
should we move these lines back a little to be inline with the remote_addr above? (or close to)